approval_polls/views.py

```diff
@@ -21,6 +21,18 @@ def index(request):
       polls = paginator.page(paginator.num_pages)
   return render(request, 'approval_polls/index.html', {'latest_poll_list' : polls})
 
+def PollsView(request):
+  poll_list = Poll.objects.filter(pub_date__lte=timezone.now(), user_id=request.user).order_by('-pub_date')
+  paginator = Paginator(poll_list, 10)
+  page = request.GET.get('page')
+  try:
+      polls = paginator.page(page)
+  except PageNotAnInteger:
+      polls = paginator.page(1)
+  except EmptyPage:
+      polls = paginator.page(paginator.num_pages)
+  return render(request, 'approval_polls/myPolls.html', {'latest_poll_list' : polls})
+
```

Comment:
As this code segment is repeated twice in this file, could you make a helper method called `getPolls` or something similar that takes a list of polls returned from the database and the template file path, and contains the functionality present between line 26 - 34?